GitUI/Theming/ThemeModule.cs

```diff
@@ -83,8 +83,10 @@ private static ThemeSettings LoadThemeSettings()
                 return CreateFallbackSettings(invariantTheme);
             }
 
-            IsDarkTheme = theme.SysColorValues[KnownColor.Window].GetBrightness() < 0.5;
-
+            if (theme.SysColorValues.ContainsKey(KnownColor.Window))
+            {
+                IsDarkTheme = theme.SysColorValues[KnownColor.Window].GetBrightness() < 0.5;
+            }
```

Comment:
Missing newline after `}` let's the build fail.
```suggestion
            }
            
```